mintpy/utils/utils1.py

```diff
@@ -86,7 +86,7 @@ def get_residual_std(timeseries_resid_file, mask_file='maskTempCoh.h5', ramp_typ
     fc = np.loadtxt(std_file, dtype=bytes).astype(str)
     std_list = fc[:, 1].astype(np.float32).tolist()
     date_list = list(fc[:, 0])
-    return std_list, date_list
+    return std_list, date_list, std_file
```

Comment:
Sounds good to me. Could you also update the related comments in the following lines?

https://github.com/insarlab/MintPy/blob/4357b8c726dec8a3f936770e3f3dda92882685b7/mintpy/utils/utils1.py#L54-L58

Reply:
The comment is added there. Thanks.